engine/src/main/kotlin/games/figureit/engine/first/gamecontrol/positiongenerator/PositionGeneratorFirstFree.kt

```diff
@@ -19,7 +19,7 @@ class PositionGeneratorFirstFree: PositionGenerator {
         throw RuntimeException("Not enough space")
     }
 
-    private fun pixelIsOccupied(testPosition: Position, positions: List<Position>): Boolean {
+    private fun pixelIsOccupied(testPosition: Position, positions: Collection<Position>): Boolean {
         for (position in positions) {
```

Comment:
why didnt you use "return positions.contains(position)"?

Reply:
sure :+1: